### DISCUSSION TITLE
Monthly trade data by HS code

### DISCUSSION THREAD

**Discussion by tortugaantigua** [eurostat/restatapi#7] [Q&A] [answered]
I'm looking to automate monthly downloads of EU trade data. The database DS-045409 seems to be the right one, as I need data by 1) individual EU reporter, 2) individual partner (not necessarily in the EU), 3) by month), and 4) by HS code. Is this database available through the restatapi package? It seems like the Comext tables are differently built than the other Eurostat tables, and I'm wondering if there's still a way to download them.

**Comment by mmatyi** [comment, marked as answer]
@tortugaantigua the COMEXT trade data does not disseminated through the standard SDMX web API so the restatapi will not be able to download for you. You have to download manually the monthly zip files following these instructions: https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?sort=1&file=comext%2Fbulk_download%2FInstructions+on+how+to+use+the+bulkdownload+facility.pdf

**Reply by tortugaantigua** [reply]
Thanks for the response! Is that functionality something that restatapi might implement in the future?

**Reply by mmatyi** [reply]
Yes that could be as a new function like get_comext_data
